### PR TITLE
Added bio to members

### PR DESCRIPTION
**Pull request by luca590**

### CODE REVIEW

**Comment by luca590** [comment]
Also removed jane and john

**Comment by tohcheryl** [comment]
blah blah blah

File: members/LucasGaylord.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 #Lucas Gaylord Bio#
 
 I listen to movie scores and code in VIM
```
Comment (phmignot) [inline members/LucasGaylord.md]: We have so much things in common :hear_no_evil:

**Review by phmignot — COMMENTED** [review]
Looking forward for the next update of your bio !

**Pull request merged**